x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/flyout/index.test.tsx

```diff
@@ -227,6 +241,46 @@ describe('Event filter flyout', () => {
     });
 
     expect(getFormEntryState(getState())).not.toBeUndefined();
-    expect(getFormEntryState(getState())?.item_id).toBe(createdEventFilterEntryMock().item_id);
+    expect(getFormEntryState(getState())?.item_id).toBe(
+      mockedApi.responseProvider.eventFiltersGetOne.getMockImplementation()!().item_id
+    );
+  });
+
+  it('should not display banner when platinum license', async () => {
```

Comment:
can you create "positive" cases for this? when the banner should be displayed.

Reply:
the last test case is about the positive one

Reply:
Yes, but I mean the create and edit by global. Ignore it if they are not necessary.

Reply:
yeah not necessary, banner ir gonna be displayed only when is initially by policy and license under platinum